src/WebdriverClassicDriver.php

```diff
@@ -155,7 +157,15 @@ public function reset(): void
 
     public function visit(string $url): void
     {
-        $this->getWebDriver()->navigate()->to($url);
+        try {
+            $this->getWebDriver()->navigate()->to($url);
+        } catch (TimeoutException $e) {
+            // The W3C compatible Selenium Server.
+            throw new DriverException('Page failed to load: ' . $e->getMessage(), 0, $e);
+        } catch (ScriptTimeoutException $e) {
```

Comment:
1. I would consider merging those two into one, since the body logic is the same:
   ```php
        } catch (TimeoutExceptio|ScriptTimeoutException $e) {
   ```
2. In other places, we typically catch all exceptions and wrap them in the driver one. I'm wondering if it wouldn't also make sense to do that here too?
   ```php
        } catch (Throwable $e) {
            throw new DriverException('Page failed to load: ' . $e->getMessage(), 0, $e);
        }
   ```

Reply:
> In what PHP version this syntax was introduced?

[7.1](https://3v4l.org/2smoG)

> I prefer using most specific exception classes to understand when WebDriver misbehaves and starts to use unexpected exception classes.

I would normally agree, but not doing that breaks the interface contract and fixing it later on is technically a BC break. 🙈 

I'm only thinking about it - I'm not fully convinced in either way.

Reply:
> ```} catch (TimeoutExceptio|ScriptTimeoutException $e) {```

Done.

> I would normally agree, but not doing it breaks the interface contract and fixing it later on is technically a BC break. 🙈 I'm only thinking about it - I'm not fully convinced in either way.

Let's keep it. WebDriver exception thrown instead of DriverException isn't covered by BC promise anyway 😄